modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -1,62 +1,36 @@
-
-|===
-| Correct | Incorrect
-
-| A static IP address has been allocated.
-| Obtain a static IP address.
-
-|===
+Prerequisites are conditions that must be in place before the user starts the procedure. If a prerequisite is a procedure or an assembly, include a link to them. See also _The IBM Style Guide_ footnoteref:[ibm-style-guide] for details on writing prerequisites.
 
-// [bhardest] - The above table illustrates the form I've used for prereqs that has been most effective. However, we might want to soften it to a suggestion if others prefer to state prereqs differently.
-
-If a prerequisite is a procedure that the user must do before starting this procedure, include a link to it.
-
-The `Prerequisites` section is optional. Try to focus on relevant prerequisites that users might not otherwise be aware of. Don't list obvious prerequisites.
-
-// [bhardest] - We'll probably need to add some examples to further illustrate the different scenarios that are described above.
+Focus on relevant prerequisites that users might not otherwise be aware of. Do not list obvious prerequisites.
 
 [[writing-the-procedure]]
 .Writing the Procedure
-The procedure consists of one or more steps required to complete the procedure. If there is just a single step, use an unnumbered bullet. If there are multiple steps, use a numbered list.
-
-// [bhardest] - While single-step procedures are allowed, it can be difficult for users to scan/find what they need if there are a large number of single-step procedures. We should advise writers how to handle this type of scenario (for example, if each procedure is a single step to run a single command). The way that I have handled this scenario in the past has been to use a two-column table in which the left column defines the procedure you can do, and the right column lists the command to use.
-
-Start each step with an active verb, because each step corresponds to one user action. Include only one command or action per step.
-
-////
-[bhardest] - It might be helpful here to include some examples that demonstrate different scenarios. For example:
-* Single-step procedure
-* Multi-step procedure (preferably showing both the command/action and "result" of the procedure)
-* Step with choices (i.e. step with indented bulleted list)
-* procedure with substeps (with some accompanying guidelines about how much step-nesting is appropriate)
-////
+The procedure consists of one or more steps required to complete the procedure. Each step describes one action.
 
-.Adding Additional Resources
-If necessary, you can provide a bulleted list of links to other concept, procedure, and reference modules that are closely related to this procedure. Only include items that are likely to be of immediate interest to a user, not every item that could conceivably be related.
+If the procedure consists of a single step only, use an unnumbered bullet instead of a numbered list.
```

Comment:
Suggestion:

"For single-step procedures, use an unnumbered bullet instead of a numbered list."